cmd/server.go

```diff
@@ -77,39 +78,34 @@ var serverCmd = &cobra.Command{
 		}
 
 		// start RPC server
-		go func() {
-			defer running.Done()
-
-			err := startRPC(
+		group.Go(func() (err error) {
+			if err = startRPC(
 				ctx,
 				getRPCAddr(),
 				sslConfig,
 				viper.GetString("root"),
 				viper.GetBool("self-serve"),
-			)
-			if err != nil {
-				log.WithError(err).Fatal("could not run RPC")
+			); err != nil {
+				return fmt.Errorf("%v: could not run RPC", err)
```

Comment:
We use `errors.Wrap` from `github.com/pkg/errors` instead of `fmt.Errorf` to avoid losing information. Could you please change it? The code here should look like `errors.Wrap(err, "could not run RPC")`. Same (but with different error message) below.